portfolio/src/main/java/com/google/sps/data/Video.java

```diff
@@ -125,4 +133,20 @@ public void setIsPublic(boolean isPublic) {
     public boolean isPublic() {
         return isPublic;
     }
+
+    // Given thumbnails, get the highest quality one.
+    public static String maxResThumbnailUrl(ThumbnailDetails details) {
+            Thumbnail[] thumbnails = new Thumbnail[]{
+                details.getMaxres(),
+                details.getStandard(),
+                details.getHigh(),
+                details.getMedium(),
+                details.getDefault()
+            };
+            Thumbnail highestRes = null;
+            for (int thumbnailInd = 0; highestRes == null; thumbnailInd ++) {
```

Comment:
Please leave a comment indicating that these thumbnails are sorted.

Reply:
Added.